doc/howto/network_ipam.md

```diff
@@ -0,0 +1,45 @@
+(network-ipam)=
+# How to display IPAM information of a LXD deployment
+
+Checking the IPAM information for your LXD setup can help you debug networking issues. You can see which IP addresses/ranges are used for instances, network interfaces, forwards, and load balancers and use this information to track down where traffic is lost.
```

Comment:
I didn't mean that you should replace the original intro, but add this as a second paragraph. I liked the intro. :)

Reply:
Is it "IP addresses" or "IP ranges" or both?

Reply:
IP addresses